### ISSUE TITLE
No license in javascript

### ISSUE THREAD

**Issue by paulberg** [angular-ui/ui-ace#65]
The javascript file does not contain any comment as to the source of the code or the license it is under. This makes it difficult for end users to see the license. Could this be added to the source files?

**Comment by douglasduteil** [comment]
Yes.

**Comment by richleland** [comment]
Looks like this is already handled by the presence of LICENSE and the license property in bower.json? Or is this adding a banner to the top of the file?

**Comment by douglasduteil** [comment]
I guess @paulberg wants a banner on the top of each dist files right ?
A little bit of grunt concat will do it :+1:

**Comment by richleland** [comment]
mattstyles/grunt-banner works well too.

**Comment by douglasduteil** [comment]
Yeah look nice but I would like to tackle the IIFE problem in the same line.
So I would be cool to have on the dist task

```js
// The banner
/* bla bla bla */

// The IIFE stuff
(function(window, document, undefined) {'use strict';

  // The src content
  angular.module...

// The final IIFE stuff
})(window, document);
```

**Comment by douglasduteil** [comment]
What you think @richleland  ?
